docs/getting-started/what-is-dotnet.md

```diff
@@ -95,15 +95,21 @@ Other libraries are installed as [NuGet](http://www.nuget.org/ "NuGet") packages
 
 ## .NET Core CLI ##
 
-.NET Core includes a new cross-platform command-line SDK known as the CLI (for Command-Line Interface). The CLI is a UNIX-friendly set of tools for building .NET Core applications. It abstracts the C# compiler, the NuGet package manager, and includes incremental build logic. It can also integrate with the .NET Native tool chain to produce high-performance native apps and libraries. The CLI allows developers to build and test applications with a minimal installation on developer, lab, or production machines. Since some IDEs, such as Visual Studio Code nad Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  
+.NET Core includes a new cross-platform command-line SDK known as the CLI (for Command-Line Interface). The CLI is a UNIX-friendly set of tools for building .NET Core applications. It abstracts the C# compiler, the NuGet package manager, and includes incremental build logic. It can also integrate with the .NET Native tool chain to produce high-performance native apps and libraries. The CLI allows developers to build and test applications with a minimal installation on developer, lab, or production machines. Since some IDEs, such as Visual Studio Code and Visual Studio, use the CLI under the covers, developers can now choose the level they want to interact with the tool chain by working with the CLI directly, using an editor that uses the CLI, or developing with an IDE that uses the CLI internally.  
 
 ## Language Support and Development Environments ##
 
-.NET Core provides a runtime environment for your app, as well as a collection of libraries whose types and members your application can use. .NET Core itself is language agnostic: it is not a programming language, but any programming languages that targets .NET Core can be used to develop .NET Core applications, and an application written in one programming language can seamlessly access types and members in libraries developed by using a different programming language. 
+.NET Core provides a runtime environment for your app, as well as a collection of libraries whose types and members your application can use. .NET Core itself is language agnostic: it is not a programming language, but any programming language that targets .NET Core can be used to develop .NET Core applications, and an application written in one programming language can seamlessly access types and members in libraries developed by using a different programming language. 
```

Comment:
.NET Core provides a runtime environment and a collection of libraries. .NET Core is language agnostic: any programming language that targets .NET Core can be used to write apps. An app written in one programming language can access types and members in libraries developed in a different programming language.